sure.py

```diff
@@ -1383,8 +1409,22 @@ def criticalCheckseventeen(cluster_health_data, system_ip, log_file_logger):
 	except Exception as e:
 		log_file_logger.exception(e)
 
+#20:Check:vManage:Validate NMS Bringup file
+def criticalChecktwenty(version):
+	#vmanage version
+	vmanage_version = float('.'.join((version.split('.'))[0:2]))
+	success = validatenmsBringup()
+	if not success:
```

Comment:
this is not handling scenario of exception  found in function validatenmsBringup
in case of exception , it will say there is a problem, when in reality, it could be file not found or other issues

Reply:
we log the exception here: log_file_logger.exception(e) 
in case of exception, the success flag will be false and we fail the test case.

Reply:
logging an exception does not mean it is handled - plus, when it crashes, the script crashes because the error was thrown (unless properly handled)

Reply:
@VeeanPrasad - where are you catching the exception? And logging the exception does us no good and give false reassurance. If it's failing, we need to know about it & provide proper response to the person running the script (e.g.:  "file not found")